gateway/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM --platform=${BUILDPLATFORM:-linux/amd64} teamserverless/license-check:0.3.9 as license-check
 
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.13 as build
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.15 as build
 
-ENV GO111MODULE=off
+ENV GO111MODULE=on
 ENV CGO_ENABLED=0
+ENV GOFLAGS=-mod=vendor
```

Comment:
This flag is not required as of 1.14

> If the vendor directory is present in the main module's root directory, it will be used automatically if the go version in the main module's go.mod file is 1.14 or higher. 

from https://golang.org/ref/mod#vendoring

Reply:
TIL, I've removed it